hypershift-operator/controllers/nodepool/nodepool_controller_test.go

```diff
@@ -988,3 +988,28 @@ func TestListMachineTemplates(t *testing.T) {
 	g.Expect(len(templates)).To(Equal(1))
 	g.Expect(templates[0].GetName()).To(Equal("template1"))
 }
+
+func TestListMachineTemplatesIBMCloud(t *testing.T) {
```

Comment:
Thanks!
I'd expect this to be just a different test case within the Test for the function, e.g
```
	testCases := struct {
		nodePool *hyperv1.NodePool
		existingTemplates
		expectedTemplateList
	}{}
```

But that's ok, we can follow up later.